src/Executor/ReleaseGroup/DiffGenerator.php

```diff
@@ -100,19 +100,30 @@ public function generateDiff(
             );
         }
 
-        $dry = $commandArgumentsTransfer->getIsDryOrFail();
-        if (!$dry) {
-            $this->prepareBranch();
-        }
+        try {
+            $dry = $commandArgumentsTransfer->getIsDryOrFail();
+            if (!$dry) {
+                $this->prepareBranch();
+            }
 
-        $this->manifestExecutor->applyManifestList([], $unappliedManifests, $inputOutput, $commandArgumentsTransfer);
+            $this->manifestExecutor->applyManifestList([], $unappliedManifests, $inputOutput, $commandArgumentsTransfer);
 
-        if ($dry) {
-            return;
-        }
+            if ($dry) {
+                return;
+            }
 
-        $this->storeDiff($releaseGroupId, $currentBranchName, $commandArgumentsTransfer->getBranchToCompareOrFail(), $inputOutput);
-        $this->gitClean($currentBranchName);
+            $this->storeDiff($releaseGroupId, $currentBranchName, $commandArgumentsTransfer->getBranchToCompareOrFail(), $inputOutput);
+            $this->gitClean($currentBranchName);
+        } catch (GitException $exception) {
+            throw new RuntimeException(
+                sprintf(
+                    'Git error %s %s %s',
+                    $exception->getCode(),
+                    $exception->getMessage(),
+                    $exception->getRunnerResult() ? implode(PHP_EOL, $exception->getRunnerResult()->getErrorOutput()) : '',
```

Comment:
pls add getOutput also. getOutput and getErrorOutput together

Reply:
++